### PR TITLE
New footer UI with Info pages loaded dynamically

### PR DESCRIPTION
**Pull request by smitpatwa**
screenshot from 2016-07-08 14-33-11

### CODE REVIEW

File: gnowsys-ndf/gnowsys_ndf/ndf/static/ndf/scss/_app_styles.scss [hunk 1]
```diff
@@ -460,50 +460,96 @@ body{
```
Comment (saavrabh) [inline gnowsys-ndf/gnowsys_ndf/ndf/static/ndf/scss/_app_styles.scss]: Colors used in footer should be accordingly  particular theme colors.Right now you have used Clix style colors.

File: gnowsys-ndf/gnowsys_ndf/ndf/templates/ndf/footer.html [hunk 2]
```diff
@@ -1,52 +1,75 @@
...
-            <li><a href="{{site.GROUPS}}">{% trans "Groups" %}</a></li>
-            <li><a href="{{site.TERMS_OF_SERVICE}}">{% trans "Terms of Service" %}</a></li>
-            <li><a href="{{site.PRIVACY_POLICY}}">{% trans "Privacy Policy" %}</a></li>
-            <li><a href="{{site.CONTRIBUTE}}">{% trans "Contribute" %}</a></li>
-            <li><a href="{{site.CONTACT}}">{% trans "Contact" %}</a></li>
+    <div class="myfooter">
+    <div class="footer-section container content-section">
+        <div class="row" style="padding-left:0px;">
+            <div class="footer_content">
+                <img class="footer-logo" src="/static/ndf/css/themes/clix/logo.svg">
```
Comment (saavrabh) [inline gnowsys-ndf/gnowsys_ndf/ndf/templates/ndf/footer.html]: Please use `{{site.logo}}` in order to get logo as per settings,Don't use static logo path.

**Comment by smitpatwa** [comment]
1. I have added the logo src in the footer to be {{site.LOGO}} 
2. To change the backgroung according to theme, I have set the background color in app_styles.scss to be same as background in header in app_styles.scss. And added the clix background for footer in clix_styles.scss. Please let me know if anything else was to be done for this.

**Comment by saavrabh** [comment]
Good,site logo and footer is fixed.One more suggestion is when you hover on any of links in footer.Color is dark blue and text is not visible.Can you change the color so that is would be easy to read even after hover? apart from this all looks fine for me.

**Comment by likecs** [comment]
Repository, Groups and Partners webpage updated and all their styling is made common. Headers issue fixed. Cards issue while zooming-in and out also fixed. Hover issues reported in footer also fixed.

File: gnowsys-ndf/gnowsys_ndf/ndf/templates/ndf/footer1.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
+{% load i18n %}
+{% load get_latest_git_hash from simple_filters %}
```
Comment (saavrabh) [inline gnowsys-ndf/gnowsys_ndf/ndf/templates/ndf/footer1.html]: Can you explain why this footer1.html template is added? We are already using footer.html for footer

Comment (smitpatwa) [inline gnowsys-ndf/gnowsys_ndf/ndf/templates/ndf/footer1.html]: footer1.html contains the old footer html. It can be ignored now.

**Comment by kedar2a** [comment]
This commit has been merged in [alpha](https://github.com/gnowledge/gstudio/tree/alpha) branch.

**Pull request merged**